Assets/MixedRealityToolkit/Inspectors/MixedRealityShaderGUI.cs

```diff
@@ -16,25 +16,25 @@ public abstract class MixedRealityShaderGUI : ShaderGUI
     {
         protected enum RenderingMode
         {
-            Opaque,
-            TransparentCutout,
-            Transparent,
-            PremultipliedTransparent,
-            Additive,
-            Custom
+            Opaque = 0,
+            Cutout = 1,
+            Fade = 2,
+            Transparent = 3,
```

Comment:
Changing Transparent from 2 to 3 may break some materials as the dev would have chosen transparent and get fade after updating (i believe the int value of the enum is what is serialized)

Reply:
I believe that's by-design. The names have changed, but the meanings/behaviors are the same both before and after `int`-wise.

`TransparentCutout` -> `Cutout`
`Transparent` -> `Fade`
`PremultipliedTransparent` -> `Transparent`

Reply:
Yes @keveleigh is correct! It’s confusing but the names have been remapped by Unity. If Unity really wanted to get pedantic the could have renamed Transparent to Translucent, but that may have been too confusing to users.